### ISSUE TITLE
Add CVSS information

### ISSUE THREAD

**Issue by themenucha** [google/osv.dev#950]

**Comment by andrewpollock** [comment]
Yep, I think given this information is available in CVE, we should populate it in the resultant OSV record.

**Comment by oliverchang** [comment]
I think this issue belongs better in osv-scanner. 

The CVSS score can be added dynamically in the osv-scanner output, based on the grouped matches. We aren't able to modify entries from other sources.

**Comment by andrewpollock** [comment]
I'm thinking we're talking about two separate uses of CVSS. 

I was thinking about including CVSS in the OSV records that are converted from CVEs.

I'm beginning to suspect based on your most recent comment, that you're talking about including CVSS scores in the output from OSV Scanner?
